internal/npm_install/README.md

```diff
@@ -0,0 +1,4 @@
+# Manual generation of index.js
+
+Incase the API of generate_build_files.ts changes to the point where you
```

Comment:
err. not sure how that would help. I'm thinking the case where you changed the arguments that starlark passes to index.js & you've made the changes to generate_build_files.ts but you can use the generated_file_test to re-build it because the starlark and the old index.js are no longer compatible